cmd/server.go

```diff
@@ -84,7 +85,7 @@ func (s *Server) securityCredentialsHandler(w http.ResponseWriter, r *http.Reque
 
 func (s *Server) roleHandler(w http.ResponseWriter, r *http.Request) {
 	remoteIP := parseRemoteAddr(r.RemoteAddr)
-	role, err := s.store.Get(remoteIP)
+	role, err := s.getRole(remoteIP)
```

Comment:
Oh I see, sorry that's the problem with the partial view in the PR. Probably was a good idea to change here as well then 👍